files/ubuntu-20.04/ec2blkdev-init

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+#
+# cloud-init remote
+#
+# chkconfig: 345 12 99
+# description: Script to call udevadm to setup any links to devices already attached at instance launch
+
+### BEGIN INIT INFO
+# Provides: ec2blkdev
+# Required-Start: $network
+# Required-Stop: $network
+# Default-Start: 2 3 4 5
+# Default-Stop: 0 1 6
+# Short-Description: ec2blkdev
+# Description: Script to call udevadm to setup any links to devices already attached at instance launch
+### END INIT INFO
+
+. /lib/lsb/init-functions
+
+prog=/bin/udevadm
```

Comment:
I think this is the only line that is different between Ubuntu18 and Ubuntu20 version of the file? Maybe we can make this a template and substitute in the different programs, but only have 1 template for both ubuntu OSs?

Reply:
Done